.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+max_line_length = 120
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Can we leave out the `.editorconfig`? You can move it to a higher directory so you can keep taking advantage of it. I’ve mine at `~/.editorconfig`. We try keeping dot files and other configuration to a minimum. The reasoning behind it is that most ofthen these configurations make it simpler for existing maintainers at the expense to new contributors who might not know what they mean. And we always try to favour contributor friendliness over maintainers needs. I know it’s not a big deal for this single file, but it adds up quickly, and things are harder to remove than they are to add :)

Reply:
Ok no problems. Actually I added it in order to confirm with other `semantic-release` repo: https://github.com/semantic-release/semantic-release/blob/caribou/.editorconfig.
Should we remove it from each repo then ?

Reply:
> Should we remove it from each repo then ?

oh yes, let’s do that 👍

Reply:
Done